packages/realtime_client/lib/src/realtime_channel.dart

```diff
@@ -32,6 +32,8 @@ class RealtimeChannel {
   @internal
   final RealtimeClient socket;
 
+  late final bool _private;
```

Comment:
I think this attribute should be documented as well like in `RealtimeChannelConfig`